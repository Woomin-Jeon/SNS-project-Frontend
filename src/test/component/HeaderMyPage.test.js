import React from 'react';
import {shallow} from "enzyme";
import {BrowserRouter} from "react-router-dom";
import MyPageHeader from "../../components/headers/HeaderMyPage";

describe('MyPageHeader', () => {
  let loginState;
  let setLoginState;
  let currentUserState;
  let setCurrentUserState;

  it('renders well', () => {
    const component = shallow(
      <BrowserRouter>
        <MyPageHeader
          loginState={loginState}
          setLoginState={setLoginState}
          currentUserState={currentUserState}
          setCurrentUserState={setCurrentUserState}
        />
      </BrowserRouter>
    );

    expect(component).toMatchSnapshot();
  });
});
