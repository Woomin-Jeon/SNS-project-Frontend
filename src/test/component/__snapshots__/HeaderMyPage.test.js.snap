// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyPageHeader renders well 1`] = `ShallowWrapper {}`;
